packages/design-tokens/src/tokens/functional/components/grid/colors.json

```
@@ -0,0 +1,12 @@
+{
+  "grid": {
+    "column": {
+      "overlay": {
+        "color": {
+          "value": "hsl(var(--base-color-scale-red-4-hsl) / 30%)",
+          "dark": "hsl(var(--base-color-scale-pink-2-hsl) / 30%)"
+        }
+      }
+    }
+  }
+}
```

```suggestion
{
  "grid": {
    "column": {
      "bgColor": {
        "overlay": {
          "value": "hsl(var(--base-color-scale-red-4-hsl) / 30%)",
          "dark": "hsl(var(--base-color-scale-pink-2-hsl) / 30%)"
        }
      }
    }
  }
}
```

pattern-variant-property-scale

---

I guess this is just used for debugging though? I'm not sure it makes sense to expose them as CSS vars if its just used for docs/demo 🤔

---

Thanks, will adjust. 

Re. using tokens, it's just for enabling color modes overrides, which the tokens do more easily without the need for custom css overrides in the stylesheet.